app/Makefile

```diff
@@ -51,13 +51,24 @@ watch-ui: compile ## Compile and Watch JS & CSS
 
 build-ui: compile ## build production ui
 
+build-ui-package: ## build production ui for the package
+	rm -rf static
+	rm -rf ui/src/env.ts
+	cp local_server_settings/env.ts ui/src/
```

Comment:
Can you put this two subfolders like we discussed? `local_settings/server`, `local_settings/ui`